### ISSUE TITLE
Wrong loss in training

### ISSUE THREAD

**Issue by wrbbb** [miraiaroha/ACAN#5]
Thank you for your work. I'm very interested in it. But when I train with the dataset you provided, the loss is Nan or 0. What's the problem?

**Comment by wrbbb** [comment]
Setting the learning rate to 2e-4 is too large. Reduce the learning rate to solve this problem.
